x/checkers/types/full_game.go

```diff
@@ -0,0 +1,42 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/xavierlepretre/checkers/x/checkers/rules"
+)
+
+type FullGame struct {
+	Creator sdk.AccAddress
+	Index   string
+	Game    rules.Game
+	Red     sdk.AccAddress
+	Black   sdk.AccAddress
+}
+
+func (fullGame *FullGame) ToStoredGame() (storedGame *StoredGame) {
+	storedGame.Creator = fullGame.Creator.String()
+	storedGame.Index = fullGame.Index
+	storedGame.Game = fullGame.Game.String()
+	storedGame.Turn = fullGame.Game.Turn.Color
+	storedGame.Red = fullGame.Red.String()
+	storedGame.Black = fullGame.Black.String()
+	return storedGame
+}
+
+func (storedGame *StoredGame) ToFullGame() (fullGame *FullGame) {
+	var err error
+	fullGame.Creator, err = sdk.AccAddressFromBech32(storedGame.Creator)
```

Comment:
nit: no need for the var
```suggestion
	fullGame.Creator, err := sdk.AccAddressFromBech32(storedGame.Creator)
```